rust/src/transformations/make_stable_expr/expr_drop_nan_or_null/test.rs

```diff
@@ -0,0 +1,42 @@
+use std::collections::HashSet;
+
+use polars::df;
+use polars::prelude::{col, IntoLazy, NamedFrom};
+use polars::series::Series;
+
+use crate::domains::{AtomDomain, LazyFrameDomain, SeriesDomain};
+use crate::measurements::make_private_lazyframe;
+use crate::measures::MaxDivergence;
+use crate::metrics::SymmetricDistance;
+use crate::polars::PrivacyNamespace;
+
+use super::*;
+
+#[test]
+fn make_expr_drop_nan_null_standard() -> Fallible<()> {
+    let series_domain = SeriesDomain::new("", AtomDomain::<f32>::default());
+    let lf_domain = LazyFrameDomain::new(vec![series_domain])?.with_margin(
+        HashSet::new(),
+        Margin::default().with_max_partition_length(5),
+    )?;
+    let lf = df!("" => &[1.0, f32::NAN])?.lazy();
+
+    let lf_filter =
+        lf.clone()
+            .select([col("").drop_nans().drop_nulls().dp().sum((0.0, 1.0), None)]);
```

Comment:
I see the NaN, but does the `drop_nulls` here actually do anything? Would it make sense to test `drop_nans` and `drop_nulls` separately?

Reply:
OpenDP assumes data could be nan or null. Dropping nulls leaves nans, and vice versa. Good point about splitting the tests. I added more tests.